src/main/java/bio/terra/service/filedata/google/firestore/FireStoreUtils.java

```diff
@@ -339,4 +337,18 @@ static boolean shouldRetry(Throwable throwable, boolean isBatch) {
       }
     }
   }
+
+  public boolean collectionHasDocuments(Firestore firestore, Query collectionQuery)
+      throws InterruptedException {
+    int docCount =
+        runTransactionWithRetry(
+            firestore,
+            (xn -> {
+              ApiFuture<QuerySnapshot> querySnapshot = xn.get(collectionQuery);
```

Comment:
Would it make sense to add the `limit(1)` in here since it's really a function of this method

Reply:
I might even exect this method to have a collection object (like the name or IDK if firestore has an object to refer to a collection)